src/net_processing.cpp

```diff
@@ -1226,10 +1226,10 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
         }
     } // release cs_main
 
-    if (it != pfrom->vRecvGetData.end()) {
+    if (it != pfrom->vRecvGetData.end() && !pfrom->fPauseSend) {
         const CInv &inv = *it;
-        it++;
```

Comment:
So inv is being discarded in master?

Reply:
Yes. But only on master, luckily @theuni caught this pre-release.